apps/betterangels-backend/accounts/models.py

```diff
@@ -118,14 +120,29 @@ class Meta:
         constraints = [models.UniqueConstraint(fields=["hmis_id", "agency"], name="unique_hmis_id_agency")]
 
 
+def phone_number_field_default_value() -> PhoneNumber:
```

Comment:
is there a reason we need this?

Reply:
You're right, it looks like we don't!

I had thought we needed a default PhoneNumber dict so the JSONField wrapper could understand how to serialize it. The tests ended up passing without this in it, so it looks like adding this does nothing of value.

Thank you!!!